Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow_DEPRECATED.cs

```diff
@@ -1,469 +0,0 @@
-/*
```

Comment:
This file should be removed - even though the scripting define conditional is weird.

Reply:
File has been restored - the solution was to change the multi-line string literal that was conflicting with the use of the `EOS_DISABLE` scripting define so that it's properly escaped.

File has been re-introduced with proper application of `EOS_DISABLE`, and this conversation is being marked as resolved.